history/characters/wc_easteregg_characters.txt

```diff
@@ -8,4 +8,312 @@
+		}
+	}
+	603.1.1 = { # Age: 25
+		trait = shrewd
+	}
+	612.1.1 = {
+		death = {
+			death_reason = death_vanished
+		}
+	}
+}
+
+easteregg5 = {
+	name = Arren
+	dna = arren_dna
+	dynasty = easteregg_dyn1 # Greenwood
+	culture = azerothian
+	religion = holy_light
+	disallow_random_traits = yes
+	#Personality
+	trait = brave
+	trait = compassionate
+	trait = stubborn
+	father = easteregg2 # Garith
+	mother = easteregg3 # Aliza
+	579.1.1 = { 
+		birth = yes
+		effect = {
+			add_character_flag = has_scripted_appearance
+		}
+		trait = creature_human
+	}
+	595.1.1 = { # Age: 16
+		trait = education_martial_4
+	}
+	603.1.1 = {
+		employer = 8 # Varian Wrynn
+	}
+	604.1.1 = { # Age: 25
+		trait = blademaster_2
+	}
+	608.1.1 = {
+		death = {
+			death_reason = death_battle
+		}
+	}
+}
+
+easteregg6 = { # Mother of Derthon and Melyina
+	name = Evelyn
+	dna = evelyn_dna
+	female = yes
+	dynasty = easteregg_dyn4 # Smith
+	culture = azerothian
+	religion = holy_light
+	disallow_random_traits = yes
+	#Personality
+	trait = deceitful
+	trait = fickle
+	trait = lustful
+	556.1.1 = { 
+		birth = yes
+		effect = {
+			add_character_flag = has_scripted_appearance
+		}
+		trait = creature_human
+		employer = 368 # Parr Ridgewell (Lakeshire)
+	}
+	571.1.1 = { # Age: 16
+		trait = education_intrigue_3
+	}
+	583.1.1 = {
+		employer = 378 # Remington Ridgewell (Lakeshire)
+	}
+	611.1.1 = {
+		death = {
+			death_reason = death_natural_causes
+		}
+	}
+}
+
+easteregg7 = {
+	name = Derthon
+	dna = derthon_dna
+	dynasty = easteregg_dyn1 # Greenwood
+	culture = azerothian
+	religion = holy_light
+	disallow_random_traits = yes
+	#Personality
+	trait = lustful
+	trait = gregarious
+	trait = fickle
+	father = easteregg2 # Garith
+	mother = easteregg6 # Evelyn
+	580.1.1 = { 
+		birth = yes
+		trait = creature_human
+		effect = {
+			add_character_flag = has_scripted_appearance
+		}
+		employer = 378 # Remington Ridgewell (Lakeshire)
+	}
+	590.1.1 = {
+		effect = {
+			set_relation_friend = character:easteregg8 # Melyina
+		}
+	}
+	596.1.1 = { # Age: 16
+		trait = education_martial_3
+	}
+	603.1.1 = {
+		employer = 8 # Varian Wrynn
+		effect = {
+			set_relation_friend = character:easteregg4 # Fenarith
+		}
+	}
+	605.1.1 = { # Age: 25
+		trait = strong
+	}
+}
+
+easteregg8 = {
+	name = Melyina
+	dna = melyina_dna
+	female = yes
+	dynasty = easteregg_dyn1 # Greenwood
+	culture = azerothian
+	religion = holy_light
+	disallow_random_traits = yes
+	#Personality
+	trait = shy
+	trait = ambitious
+	trait = stubborn
+	father = easteregg2 # Garith
+	mother = easteregg6 # Evelyn
+	582.1.1 = { 
+		birth = yes
+		effect = {
+			add_character_flag = has_scripted_appearance
+		}
+		trait = creature_human
+		employer = 378 # Remington Ridgewell (Lakeshire)
+	}
+	598.1.1 = { # Age: 16
+		trait = education_martial_2
+	}
+	603.1.1 = {
+		employer = 8 # Varian Wrynn
+	}
+	613.1.1 = {
+		death = {
+			death_reason = death_vanished
+		}
+	}
+}
```

Comment:
They are born outside of the marriage, why they're not labeled bastards?

Reply:
> They are born outside marriage, why they're not labeled bastards?

Derthon was my own character in WoW and Melyina was my friend's. We roleplayed on them but never made a point of them being bastards, although I guess technically they would have been. I can change it.

Reply:
Fixed in 89df74832164f9b4498d3f14e2c00f3ec4913b9b.